data/hai/hai missions.txt

```diff
@@ -209,6 +209,79 @@ mission "Ask the Hai about the Unfettered"
+		conversation
+			`As you sit in a spaceport bar waiting for the food that you've ordered, a Hai unceremoniously pulls up a chair opposite you and takes a seat. He looks to be a merchant, and one who has seen more than a few things besides. He squints at you before speaking, "You are Captain <last>, are you not?"`
+			choice
+				`	"That would be me."`
+					goto acknowledge
+				`	"I think you have the wrong captain."`
+			`	He slaps a device on the table, and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`
+			label acknowledge
+			`	He heaves a sigh. "You've been fighting the Unfettered, you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers, and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
+			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see, because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
+			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, so you might notice that some people are less offended when you shoot down someone they've been arguing with, but everyone has friends on all sides, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you. Despite that, the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first..."`
+			`	He looks at you very seriously to make sure what he's said has sunk in.`
+			choice
+				`	"I understand, I've stepped on some toes."`
+					goto trouble
+					to display
+						"reputation: Hai" < 1
+				`	"I understand, thank you for warning me."`
+					goto end
+					to display
+						"reputation: Hai" >= 1
+				`	"How could I be in trouble for defending myself against Sympathizers?"`
+					to display
+						"reputation: Hai" < 1
+				`	"Why should I get in trouble for defending myself against Sympathizers?"`
+					to display
+						"reputation: Hai" >= 1
+			action
+				set "Hai Sympathizer Warning: Explained Trouble"
+			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
```

Comment:
```suggestion
			`	He shrugs. "You shouldn't try to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
```

Reply:
Based on the wiki of Hai speech:
`They make occasional grammatical errors, particularly leaving out articles or inserting the where they are not needed. (This is to hint that they speak a language like Latin that does not use articles.)`
I generally try and place the negative after the action when random Hai are speaking.

"You shouldn't try to" and "You should try not to" are functionally interchangeable in English, but they do have subtly distinct emphasis, and that emphasis is pronounced in Latin.

In this case he's not just saying "don't try and apply this to us", he's telling you to actively put effort into trying *not* to apply those expectations.
It's not "don't think about it", it's saying "deliberately think about the opposite".